backend/src/controller/chatController.ts

```diff
@@ -90,7 +90,7 @@ async function handleChatWithoutDefenceDetection(
 	chatHistory: ChatHistoryMessage[],
 	defences: Defence[]
 ): Promise<LevelHandlerResponse> {
-	const updatedChatHistory = createNewUserMessages(message, null).reduce(
+	const updatedChatHistory = createNewUserMessages(message, undefined).reduce(
```

Comment:
You don't need to explicitly pass `undefined` here, just call the function with one arg.